boards/pyruler/4.x/code.py

```diff
@@ -78,23 +79,54 @@ def type_alt_code(code):
         leds[i].value = c
     if caps[0]:
         if ENABLE_KEYBOARD:
-            if WINDOWS_COMPUTER:
+            if WINDOWS:
                 type_alt_code(234)
-            else:
+            elif MAC:
                 kbd.send(Keycode.ALT, Keycode.Z)
+            else:
```

Comment:
`elif OS == LINUX:`

Reply:
If OS isn't set properly, no keycode will be sent

Reply:
Empty `else:` at the end of the block was causing it to not work. Just leaving it out works fine.